Cargo.toml

```diff
@@ -36,3 +40,80 @@ gtk_3_16 = ["gtk_3_10", "gtk/v3_16"]
 gtk_3_18 = ["gtk_3_16", "gtk/v3_18"] #for CI tools
 gtk_3_20 = ["gtk_3_18", "gtk/v3_20"] #for CI tools
 gtk_3_22 = ["gtk_3_20", "gtk/v3_22"] #for CI tools
+futures = ["futures-core", "futures-executor", "futures-channel", "futures-util", "glib/futures", "gio/futures"]
```

Comment:
Maybe here also must be some `gtk_3_xxx` ?

Reply:
No GTK is needed here

Reply:
Ok

Reply:
`"glib/futures"` IMHO unneeded here, as it included in `"gio/futures"`